### ISSUE TITLE
[UPDATE] Dropper - MINIGAME

### ISSUE THREAD

**Issue by TrustedMercury** [Derkades/Minigames#134]
Add more maps, more maps, more fun 👍

In the JUNGLE and RAINBOW map, make it slightly harder such that people die when they fall and can't just walk.

Don't make the maps too hard.

**Comment by Derkades** [comment]
rainbow map will be impossible without walking I think

**Comment by TrustedMercury** [comment]
No, rainbow and the wood one is the only I can win, as I can literally walk halfway.

**Comment by Derkades** [comment]
I mean when walking is *not* possible, most of the maps will be unplayable. We need to tweak the maps or build new maps before disabling walking
